.devcontainer/entrypoint.sh

```diff
@@ -44,13 +44,6 @@ if [ "$1" = "--build-engines" ]; then
     shift
 fi
 
-if [ "$1" = "--editable" ]; then
```

Comment:
@eliteprox I'm fine with keepign this if you think it is usefull.

Reply:
I don't think it will be needed outside of devcontainer scenarios. This is good.